rpcd/playbooks/roles/rpc_maas/templates/elasticsearch_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
```

Comment:
Leaving this as-is, because the style matches the other templates. For what that's worth.